aws-iam-role/variables.tf

```diff
@@ -35,11 +35,25 @@ variable "max_session_duration" {
   default     = 3600
 }
 
+// DEPRECATED: use saml_idps instead
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Single line comments should begin with #